pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,332 @@
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/vaccine_wide-479daadfcb96b7c8a0b614d7aab152d43018e877.jpg?s=600"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/music-videos/">
+              <h2>Music Videos</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/04/942186930/dua-lipa-tiny-desk-home-concert"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20"
+                >Dua Lipa: Tiny Desk (Home) Concert</amp-fit-text
+              >
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/dua_wide-41767488aed1f75828f980b6b41339a0c65d4964.jpg?s=600"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/national/">
+              <h2>National</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/03/942034617/time-names-its-kid-of-the-year-water-testing-scientist-gitanjali-rao"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                'Time' Names Its Kid Of The Year: Water-Testing Scientist
+                Gitanjali Rao
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/time201207_koty.coverfinal_wide-a7cff22dc41f55823d394817f2c1a0acfc105a66.jpg?s=600"
+            layout="intrinsic"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/book-reviews/">
+              <h2>Book Reviews</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/04/941982629/maureen-corrigans-10-books-that-will-connect-you-in-a-socially-distant-year"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                Maureen Corrigan's 10 Books That Will Connect You In A Socially
+                Distant Year
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/corrigan2020_wide-26e0b93fd657db854e1ba2bdb7b18539ca72ecfa-s1100-c85.jpg"
+          >
+        </article>
+      </amp-base-carousel>
+    </div>
+  </body>
+</html>
+```
+
+See the [demo in action on Glitch](TODO).
+
+# Managing events
```

Comment:
I find this heading a bit confusing as it doesn't really explain what this section is about. Maybe: "Interacting with Bento components. 

This section should also move into the "Use Bento components in non-AMP pages" section.